### PR TITLE
Update actions, documentation templates and configuration files to v0.1

### PR DESCRIPTION
**Pull request by juanpedromoreno**
> ❕This is an automated PR created from a release in [`47degrees/.github`](https://github.com/47degrees/.github).

# What has been done in this PR?

Update Github Actions workflows, documentation templates and configuration files to v0.1.

# Included changes

- Adds repos list
- Adds GH worflows
- Adds default GH files and workflows

- First version of the documentation files, including AUTHORS.md, CODE_OF_CONDUCT.md, CONTRIBUTING.md and NOTICE.md
- First version of the workflow files, including workflows for updating changelog and docs, for running the release task and executing the ci.

In order for running ci, docs and release workflows the following aliases (with the same, or different commands) should be created in build.sbt:

```
addCommandAlias("ci-test", "fix --check; mdoc; test; publishLocal; all scripted")
addCommandAlias("ci-docs", "mdoc; headerCreateAll")

//Not needed if using `sbt-ci-release`
addCommandAlias("ci-release", "publish")
```

### CODE REVIEW

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      - master
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update release draft
+        uses: release-drafter/release-drafter@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.ADMIN_GITHUB_TOKEN }}
```
Comment (BenFradet) [inline .github/workflows/release-drafter.yml]: :fire:

File: .gitignore [hunk 2]
```diff
@@ -1,16 +1,69 @@
...
-# PGP keys
-secret_key.asc
+### sbt-fix ###
+# Scalafix & Scalafmt configuration files are automatically downloaded by `sbt-fix`
+# https://github.com/47degrees/sbt-fix
 
-.metals/
-project/.bloop/
+.scalafix.conf
+.scalafmt.conf
```
Comment (BenFradet) [inline .gitignore]: 💬

File: .gitignore [hunk 3]
```diff
@@ -1,16 +1,69 @@
...
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
 
-# PGP keys
-secret_key.asc
+### sbt-fix ###
+# Scalafix & Scalafmt configuration files are automatically downloaded by `sbt-fix`
+# https://github.com/47degrees/sbt-fix
```
Comment (juanpedromoreno) [inline .gitignore]:
```suggestion
```

File: .gitignore (hunk 2 above)
Comment (juanpedromoreno) [inline .gitignore]:
```suggestion
```

**Pull request merged**